src/olympia/editors/tests/test_models.py

```diff
@@ -594,15 +609,10 @@ def test_all_users_by_score(self):
         self._give_points(status=amo.STATUS_PUBLIC)
         self._give_points(user=user2, status=amo.STATUS_NOMINATED)
         users = ReviewerScore.all_users_by_score()
-        assert len(users) == 2
-        # First user.
+        assert len(users) == 1
         assert users[0]['total'] == 200
         assert users[0]['user_id'] == self.user.id
         assert users[0]['level'] == amo.REVIEWED_LEVELS[0]['name']
-        # Second user.
-        assert users[1]['total'] == 120
-        assert users[1]['user_id'] == user2.id
-        assert users[1]['level'] == ''
```

Comment:
This is removing the testing of having multiple users and sorting, so add `user2` to the reviewer group.